packages/salmon-lambda-functions/src/generic/slack.ts

```diff
@@ -1,12 +1,21 @@
+import { Bech32 } from '@defichain/jellyfish-crypto'
 import { OraclesManager } from '@defichain/salmon-oracles-functions'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import BigNumber from 'bignumber.js'
 import fetch from 'node-fetch'
 import { EnvironmentConfig } from './environment'
 
 export async function checkBalanceAndNotify (oraclesManager: OraclesManager, env: EnvironmentConfig): Promise<void> {
-  const balance = await oraclesManager.getBalance(env.oceanUrl, env.network)
-  if (balance.lt(2.0)) {
-    const address = await oraclesManager.getAddress()
+  const pubKey = await oraclesManager.walletAccount.publicKey()
+  const address = Bech32.fromPubKey(pubKey, 'df')
```

Comment:
https://github.com/DeFiCh/jellyfish/blob/fb4cde0000fbef9990059f04179fdc85fe86ac65/packages/jellyfish-wallet/src/wallet_account.ts#L24-L27

WalletAccount comes with `getAddress()` interface

```suggestion
  const address = await oraclesManager.walletAccount.getAddress()
```

Reply:
oh dang! thanks for pointing that out